frontend/compositions/dashboardHeader/dashboardHeader.module.css

```diff
@@ -68,38 +68,25 @@
   margin: var(--size24) var(--size24) var(--size32);
 }
 
-.desktopNav {
-  display: none;
-}
-
-.button {
+.buttonContainer button {
```

Comment:
As there's nothing else in this container I think it might be better to apply these directly on the button itself instead of nesting! It would allow us to get rid of an unnecessary `div` in the template code!

Reply:
I deleted the button container all together, but still ended up nesting the button within the larger `nav` container - I was having some issues figuring out how to target the button directly when using the global `primaryButton` class, and will continue to work on this.